README.md

```diff
@@ -101,33 +88,29 @@ ramaze                    1936.36       10.44%
 ```
 Framework       Allocs/Req Memsize/Req
 --------------------------------------
-rack                    38        3256
```

Comment:
Same as above comment. Do not update README numbers. Those will be updated once the changes are merged.

Thank you.